include/smack/BoogieAst.h

```diff
@@ -113,6 +114,17 @@ class BvLit : public Expr {
   void print(std::ostream& os) const;
 };
 
+class FpLit : public Expr {
+	bool neg;
+	std::string sig;
+	std::string expo;
+	unsigned sigSize;
+	unsigned expSize;
+public:
+	FpLit(bool n, std::string s, std::string e, unsigned ss, unsigned es) : neg(n), sig(s), expo(e), sigSize(ss), expSize(es) {}
```

Comment:
You are using tabs for indentation here. Change that to two spaces instead. You should never use tabs.